<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Laboratory website">
    <meta name="author" content="Josh Herr">
    <link rel="shortcut icon" href="ico/favicon.png">

    <title>The Herr Laboratory</title>

    <!-- Bootstrap core CSS -->

    <link href="css/bootstrap.css" rel="stylesheet">
    <link href="css/main.css" rel="stylesheet">
    <link href="css/academicons.css" rel="stylesheet">
    <link href="css/font-awesome.min.css" rel="stylesheet">
    <!--<link href="css/animate-custom.css" rel="stylesheet">-->
    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic' rel='stylesheet' type='text/css'>
    <link href='http://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>

    <script src="js/jquery.js"></script>
	<script type="text/javascript" src="js/modernizr.custom.js"></script>
	<script src="js/bootstrap.js"></script>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="assets/js/html5shiv.js"></script>
      <script src="assets/js/respond.min.js"></script>
    <![endif]-->
  </head>

  <body data-spy="scroll" data-offset="0" data-target="#navbar-main">
  	<div id="navbar-main">
      <!-- Fixed navbar -->
    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
            <span class="icon icon-shield" style="font-size:30px; color:#3498db;"></span>
          </button>
          <a class="navbar-brand hidden-xs hidden-sm" href="#home"><span class="fa fa-leaf" style="font-size:18px; color:#3abd26;"></span></a>
        </div>
        <div class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
            <li><a href="#" class="smoothScroll"><span style= "font-size:20px; font-weight: 700; color:red">The Herr Lab</a></li>
			<li> <a href="#about" class="smoothScroll"><b>Research</b></a></li>
			<!--<li> <a href="#teaching" class="smoothScroll"><b>Teaching</b></a></li>-->
			<li> <a href="#people" class="smoothScroll"><b>People</b></a></li>
			<li> <a href="#publications" class="smoothScroll"><b>Publications</b></a></li>
			<li> <a href="http://cymeandcystidium.com" class="smoothScroll"><b>Josh's Blog</b></a></li>
			<li> <a href="#news" class="smoothScroll"><b>Outreach</b></a></li>
			<!--<li> <a href="#contact" class="smoothScroll"><b>Contact</b></a></li>--></ul>
		  <ul class="nav navbar-nav navbar-right">
	      	<li class="dropdown">
	      	<a href="#" class="dropdown-toggle" data-toggle="dropdown" style= "color:red"><b>UNL</b><span class="caret"></span></a>
            <ul class="dropdown-menu" role="menu">
            <li><a href="http://www.unl.edu/"><b>University of Nebraska - Lincoln</b></a></li>
			<li><a href="http://plantpathology.unl.edu/"><b>Department of Plant Pathology</b></a></li>
			<li><a href="http://www.unl.edu/psi/"><b>Center for Plant Science Innovation</b></a></li>
			<li><a href="http://microbiology.unl.edu/microbiology-unl"><b>Microbiology at UNL</b></a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </div>
    </div>


    <div class="headerj">
          <h1 id="labname">
            The Exelixis Lab
          </h1>
          <br>
          <p id="labnameexp">
              Enabling Research in Evolutionary Biology <br>

          </p>
    </div>



    <!--<div class="container-fluid"> -->
    <div class="container">
      <div class="row-fluid toptop">
	<div class="span12">
        <h1>Outreach activities</h1>
	</div>
      </div><!--/row-->


      <div class="container">
	<h2>Videos</h2>
	<div class="row-fluid toptop">

          <div class="span6 bd-2">
            <p>A video of our PhD student Lucas who won the <a href="https://www.facebook.com/heidelberg.science.slam?pnref=story">Heidelberg science slam</a> competition in Heidelberg
	      with this talk.</p>
	    <iframe src="./outreach_files/oNenj69oy00.html" frameborder="0" allowfullscreen=""></iframe>


	  </div>



	</div><!--/row-->
	<div class="row-fluid toptop">

          <div class="span6 bd-2">
            <p>A TV report in English about our recent paper in <a href="http://www.sciencemag.org/content/346/6210/763">Science</a> that was the first outcome of the
	    1KITE project.</p>
	    <iframe src="./outreach_files/Ldak-GlHEZc.html" frameborder="0" allowfullscreen=""></iframe>


	  </div>

	  <div class="span6 bd-2">
            <p>A TV report by the nano technology magazine of 3Sat about the work of our group in the framework of the <a href="http://www.1kite.org/">1KITE</a> project.</p>

	    <iframe src="./outreach_files/CFsIBdUBFek.html" frameborder="0" allowfullscreen=""></iframe>
	  </div>

	</div><!--/row-->

	<div class="row-fluid toptop">

          <div class="span6 bd-2">
            <p>Alexis gave a <a href="http://webcast.informatik.kit.edu/74.php">video
		interview </a>(in German) about his new duties as professor of
	      computer science at KIT in conjunction with his position at HITS.</p>

	    <iframe src="./outreach_files/_6Laa58M-vk.html" allowfullscreen="" frameborder="0"></iframe>
	  </div>

	  <div class="span6 bd-2">
	    <p>Local Cretan TV (Kriti TV) report about 2012 summer school on
	      computational molecular evolution sponsored by EMBO and co-sponsored by
	      HITS.</p>

	    <iframe src="./outreach_files/v7d2VnRmnek.html" allowfullscreen="" frameborder="0"></iframe> <br>

	  </div>

	</div><!--/row-->

		<div class="row-fluid toptop">

          <div class="span6 bd-2">
	    <p>	  Local Cretan TV (Kriti TV) report about 2010 summer school on
	      computational molecular evolution sponsored by EMBO:</p>

	    <iframe src="./outreach_files/qNC-hE-6jmU.html" frameborder="0" allowfullscreen=""></iframe>



	  </div>

	  <div class="span6 bd-2">
            <p>Agniezska Borowska from HITS produced a very nice video about
	      phylogenetic inference:</p>
	    <iframe src="./outreach_files/WTRgT763P5w.html" allowfullscreen="" frameborder="0"></iframe>

	  </div>

	</div><!--/row-->
      </div>




      <div class="container">
	<h2>Newspapers and press releases</h2>
	<div class="row-fluid toptop">




	  <div class="span6 bd-2">
	    <p>An article about our yet unpublished work on identifying mislabeled sequences in the
	      <a href="http://www.the-scientist.com/?articles.view/articleNo/41821/title/Mistaken-Identities/">New Scientist</a>.</p>

	  </div>


	</div><!--/row-->

	<div class="row-fluid toptop">
	   <div class="span6 bd-2">
	    <p>Links to various press releases and articles about our <a href="http://www.sciencemag.org/content/346/6210/763">Science paper</a> on insect evolution</p>

	    <ul>
	      <li><a href="http://sco.h-its.org/exelixis/resource/outreach-material/20141107_1KITE_Science_in_RNZ.PNG">Local newspaper in the Rhein Neckar region</a></li>
	      <li><a href="http://www.gauss-centre.eu/SharedDocs/Meldungen/GAUSS-CENTRE/EN/2014/news_SuperMUC_1KITE.html">Gauss Center for Supercomputing press release</a></li>
	      <li><a href="http://sco.h-its.org/exelixis/resource/outreach-material/KIT.pdf">KIT press release</a></li>
	      <li><a href="http://www.h-its.org/en-sco/in-science-scientists-resolve-the-evolution-of-insects/">HITS press release</a></li>
	      <li><a href="http://www.3sat.de/page/?source=/nano/natwiss/175194/index.html">German-Swiss-Austrian TV channel 3Sat</a></li>
	      <li><a href="http://www.1kite.org/news.html#1kitepressreleases">Official 1KITE project homepage</a></li>
	      <li><a href="http://www.sciencefriday.com/segment/11/07/2014/piecing-together-the-puzzle-of-insect-evolution.html">Science Friday report</a></li>
	      <li><a href="http://www.wienerzeitung.at/themen_channel/wissen/natur/699537_480-Millionen-Jahre-alter-Stammbaum.html">Vienna Newspaper</a></li>
	      <li><a href="http://www.br.de/mediathek/video/sendungen/nachrichten/insektenforschung-100.html">Bavarian federal state television report: Bayerischer Rundfunk</a></li>
	    </ul>
	  </div>
          <div class="span6 bd-2">
	    <p>
            Alexis gave a <a href="http://www.kathimerini.gr/4dcgi/_w_articles_mel3_1_09/12/2012_473657">newspaper interview</a>
	    to the Greek newspaper Kathimerini about the crisis and Greek
	    researchers abroad. The drawing below by Titina Xalmatzi was also part
	    of that interview. Here is a rough <a href="http://sco.h-its.org/exelixis/resource/doc/Interview.pdf">German Translation</a> (due to lack of more time to do it properly).</p>

	    <img src="./outreach_files/AlexiPortrait.jpg"><br>

	  </div>


	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	    <p>An article about the work of our group was published in a <a href="http://sco.h-its.org/exelixis/resource/doc/RNZ_Stamatakis.pdf">local newspaper</a>.</p>
	  </div>

	  <div class="span6 bd-2">
           <p>Press releases on big plant tree with 55,000 species <a href="http://www.h-its.org/en-sco/the-tree-of-life-when-do-plants-spin-off-new-species/">English</a>
	     <a href="http://www.h-its.org/de-sco/der-stammbaum-des-lebens-wann-entwickeln-pflanzen-neue-arten/">German</a></p>
	  </div>
	</div><!--/row-->


	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	   <p> A very readable and easy to understand article was published in <a href="http://www.genomeweb.com/informatics/hpc-phylogenetics">Genome
	       Web</a>. It addresses High Performance Computing for Phylogenetics and
	     some of the work conducted at the Exelixis Lab.</p>
	  </div>

	  <div class="span6 bd-2">
            <p>Our work is briefly mentioned on pages 68-69 in the <a href="http://www.dfg.de/jahresbericht/download/dfg_jb2009.pdf">2009
		yearbook of the German Science Foundation</a> (DFG)</p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	   <p>Elisa Loza wrote a vrey nice and understandable<a href="http://www.software.ac.uk/blog/2012-08-28-evolution-farm-tree">
	       blog post</a> about RAxML and the evolutionary placement algorithm.</p>
	  </div>

	  <div class="span6 bd-2">
            <p>A <a href="http://portal.mytum.de/pressestelle/pressemitteilungen/news_article.2010-04-15.0730120084">press
		release</a> (German only) was published jointly by TUM/LRZ on the
	      integration of RAxML into the SPEC MPI 2007 benchmark.</p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	   <p>Disentangling the evolution of metazoa: Here
	     is a <a href="http://news.brown.edu/pressreleases/2009/09/bilateral">press
	       release</a> by Brown University in English and a <a href="http://portal.mytum.de/pressestelle/pressemitteilungen/news_article.2009-10-29.0458923297">press
	       release</a> by TU Munich in German describing some recent work in
	     the
	     area of phylogenomics. The Exelixis Lab (Michael Ott and Alexandros
	     Stamatakis) significantly
	     contributed to this project by providing and adapting the
	     software, organizing the computational resources, and conducting the
	     large-scale inferences on a IBM BlueGene/L supercomputer system at the
	     San Diego Supercomputer Center.</p>
	  </div>

	  <div class="span6 bd-2">
            <p>A <a href="http://www.informatik.kit.edu/309_6333.php">news item</a>
	      about Alexis appointment at the Karlsruhe Institute of Technology</p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">


	  <div class="span6 bd-2">
            <p>A <a href="http://www.h-its.org/en-sco/bioinformatics-new-type-of-bacteria-found-in-the-human-gut/">news item</a>
	      about the recent <i>Nature Methods</i> paper with the Bork lab at EMBL.
	      In this paper we apply our Evolutionary Placement Algorithm to disentangle and quantify bacterial diversity.
	    </p>
	  </div>
	</div><!--/row-->


      </div>


      <div class="container">
	<h2>Outreach Material and Links</h2>
	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	    <p>
	      A nice <a href="http://www.br.de/fernsehen/br-alpha/sendungen/alpha-forum/arndt-bode-100.html">TV interview</a> in German by Alexis' PhD advisor Arndt Bode on supercomputing in Munich.
            </p>
	  </div>
	   <div class="span6 bd-2">
	    <p>
	      An excellent collection of outreach videos
	      describing why evolution (and as a consequence the tools and software
	      we are developing) is important in many areas of our daily life: <a href="http://www.evolution-of-life.com/en/observe.html">Evolution of
		Life Web-Site</a>
            </p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	    <p>
	     A radio interview with Bernhard Misof (in German) about the <a href="http://www.1kite.org/">1KITE</a> project (1000 insect
	     transcriptome sequencing project): <a href="http://www.ndr.de/info/audio104625.html">STREAM</a><br>
	     Also have a look at two local newspaper articles about 1 Kite from <a href="http://sco.h-its.org/exelixis/resource/doc/Abendblatt_1KITE.pdf">Hamburg</a> and the <a href="http://sco.h-its.org/exelixis/resource/doc/RNZ_1KITE.pdf">Rhein-Neckar Region</a>
            </p>
	  </div>
	   <div class="span6 bd-2">
	    <p>
	      Some slides from an outreach talk on phylogenetics (in German): <a href="http://sco.h-its.org/exelixis/resource/doc/HitsOpenHouse.pdf">PDF</a> given by Alexis in May 2011.
            </p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span6 bd-2">
	    <p>
	     Our colleague in Munich Gert Wörheide hunting living fossils on German
	     Public TV (German only): <a href="http://www.zdf.de/ZDFmediathek/beitrag/video/1047484/Korallenriffe---Bedrohte-Unterwasserwelt#/beitrag/video/1047484/Korallenriffe---Bedrohte-Unterwasserwelt">Video</a>.
	     More information about this project can be found <a href="http://www.deepdownunder.de/">here</a>
            </p>
	  </div>
	   <div class="span6 bd-2">
	    <p>
	      Together
	      with Susanne Renner Alexis gave an outreach talk on phylogenetic
	      reconstruction within the framework of the Munich Science Days (in October 2008) at the
	      LMU Munich which are open to the broad public. The slides of this talk
	      are available here for download as <a href="http://sco.h-its.org/exelixis/resource/doc/StamatakisRenner2008.pdf">PDF</a>.
            </p>
	  </div>
	</div><!--/row-->

      </div>


      <div class="container">
	<h2>Geek Stuff</h2>
	<div class="row-fluid toptop">
          <div class="span12 bd-2">
	    <h3>Tree Building Outreach tool</h3>
	    <p>
	      Tree reconstruction visualization outreach tool.
	      Matthias Berger, the brother of our former PhD student Dr. Simon Berger
	      has designed a visualization tool that displays the tree building
	      phases of RAxML:
	      </p><ol>
		<li>stepwise addition order algorithm</li>
		<li>SPR moves</li>
		<li>branch length optimiziation</li>
	      </ol>
	      The tool requires JAVA and can be downloaded <a href="http://sco.h-its.org/exelixis/php/outreach.php">here</a>.

	      To execute the code, download it, unccompress it and type:<br>
	      <br>
	      <span style="font-weight: bold;">java -jar treevis.jar --tracefile test-data/combined_log.txt</span><br>
	      <br>
	      or<br>
	      <br>
	      <span style="font-weight: bold;">java -jar treevis.jar --tracefile test-data/dump_49.txt</span><br>
	      <br>
	      The input files are log files generated from real RAxML runs. You can see a screenshot below.<br>
	      <br>
	      <img src="./outreach_files/outreachVis.jpg"><br>
            <p></p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span12 bd-2">
	    <h3>RAxML on Android</h3>
	    <p>
	      Get the new RAxML code (the main code developed by our lab ) for your Android phone <a href="https://github.com/mslacken">here </a><br>
	      <br>
	      It doesn't serve any purpose except for impressing your colleagues and
	      friends by being an early technology adopter and running a scientific
	      code on your smartphone, screenshots below:<br>
	      <br>
	      <img src="./outreach_files/s1.jpg"><br>
            </p>
	  </div>
	</div><!--/row-->

	<div class="row-fluid toptop">
          <div class="span12 bd-2">
	    <h3>What does the lab actually do?</h3>
	    <p>
	      Our main focus is on developing better models and faster software such
	      that biologists can analyze their constantly growing datasets more
	      quickly and more accurately. Hence, we spend quite a large fraction of
	      our time writing code (mostly in C and C++ but occasionally also in
	      JAVA, ruby, or perl). We are all big fans of open source code and
	      provide everything we code up for free under the GNU GPL open source
	      license.<br>
	      <br>
	      To write and share sooftware (code) we use web-based tools that have a
	      code version control, that is, if we implement a new bug rather than a
	      feature (which happens more frequently than you would think) we can
	      actually roll-back and re-start from a working code (a less buggy
	      code). <br>
	      <br>
	      Typically, we use the GIT version control system that is available free
	      of charge as a web-service on platforms such as github. Those platforms
	      can be used free of charge for public open source code projects. Below
	      you will find a list of links to the github starting pages of some lab
	      members such that you can see and follow live what we are doing:<br>
	      <br>
	      <a href="https://github.com/stamatak/">Alexis</a><br>
	      <a href="https://github.com/aberer">Andre</a><br>
	      <a href="https://github.com/sim82">Simon</a><br>
	      <a href="https://github.com/zhangjiajie">Jiajie</a><br>
	      <a href="https://github.com/xflouris">Tomas</a><br>
	      <a href="https://github.com/fizquierdo">Fernando</a><br>
	      <br>

            </p>
	  </div>
	</div><!--/row-->


      </div>




      <hr>


      <footer>
        <p>© The Exelixis Lab 2013 | Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg</p>
      </footer>

    </div><!--/.fluid-container-->

    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="./outreach_files/jquery.js"></script>
    <script src="./outreach_files/bootstrap.js"></script>


</body></html>